pages/docs/community/styleguide/inclusive-language.md

```diff
@@ -0,0 +1,57 @@
+# Style Docs - Inclusive Language:
+
+While writing documentation we are indirectly reaching out to thousands of people all around the globe. 
+Thus, we must ensure that our diverse audience can connect with our information. 
+
+### 1. Culturally Inclusive Language - 
```

Comment:
there should not be any need of numbering the sections as you are already using headings `###` for the same purpose
```suggestion
### 1. Culturally Inclusive Language
```
you can remove hyphens too